content/en/blog/_posts/2021-03-30-Kubernetes-Multi-Tenancy-Usage-Models.md

```diff
@@ -0,0 +1,80 @@
+
+With this model, tenants share cluster-wide resources like ClusterRoles and CustomResourceDefinitions (CRDs) and hence cannot create or update these cluster-wide resources.
+
+The [Hierarchical Namespace Controller (HNC)](https://kubernetes.io/blog/2020/08/14/introducing-hierarchical-namespaces/) project makes it easier to manage namespace based multi-tenancy by allowing users to create additional namespaces under a namespace, and propagating resources within the namespace hierarchy. This allows self-service namespaces for tenants, without requiring cluster-wide permissions.
+
+The [Multi-Tenancy Benchmarks (MTB)](https://github.com/kubernetes-sigs/multi-tenancy/tree/master/benchmarks) project provides benchmarks and a command-line tool that performs several configuration and runtime checks to report if tenant namespaces are properly isolated and the necessary security controls are implemented.
+
+### Clusters-as-a-Service
+
+With the *clusters-as-a-service* usage model, each tenant gets their own cluster.  This model allows tenants to have different versions of cluster-wide resources, such as CRDs.
+
+The tenant clusters may be provisioned using projects like [Cluster API (CAPI)](https://cluster-api.sigs.k8s.io/) where a management cluster is used to provision multiple workload clusters. A workload cluster is assigned to a tenant and tenants have full control over cluster resources. Note that in most enterprises a central platform team may be responsible for managing required add-on services such as security and monitoring services, and for providing cluster lifecycle management services such as patching and upgrades. A tenant administrator may be restricted from modifying the centrally managed services and other critical cluster information. 
+
+In a variation of the *clusters-as-a-service* model, the tenant cluster may be a **virtual cluster**. As with other forms of virtualization, users of a virtual cluster see no significant differences between a virtual cluster and other Kubernetes clusters. However, a virtual cluster shares worker node resources and some workload state independent control plane components, like the scheduler. Other workload aware control-plane components, like the API server, are created on a per-tenant basis and additional components are used to synchronize and manage state across virtual clusters and the underlying shared cluster. This provides each tenant their own virtual control-plane that can operate independently of other tenants.
+
+The [Virtual Cluster](https://github.com/kubernetes-sigs/multi-tenancy/tree/master/incubator/virtualcluster) project implements this model, where a supercluster is shared by multiple virtual clusters. The [Cluster API Nested](https://github.com/kubernetes-sigs/cluster-api-provider-nested) project is extending this work to conform to the CAPI model, allowing use of familiar API resources to provision and manage virtual clusters.
+
+## Summary
```

Comment:
It would be nice to add a comment (or mini-section) about switching models. I.e., Say it's ok not to be sure which one you want when you're starting out, but by following a few basic rules (e.g. don't use the "default" namespace) you can ensure that you can switch from one model to another in the future with relatively little pain.

Reply:
Added this to the summary:

> In many cases, organizations will use both *namespaces-as-a-service* and *clusters-as-a-service* to address different use cases, and as different product and development teams will have varying needs. Following security and management best practices, such as not using the `default` namespace, makes it possible to switch from one model to another in the future.

Reply:
+1